main.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2021 The Volcano Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/spf13/pflag"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	cliflag "k8s.io/component-base/cli/flag"
+	"k8s.io/klog"
+
+	"volcano.sh/apis/pkg/client/clientset/versioned"
+
+	"volcano.sh/resource-exporter/pkg/args"
+	"volcano.sh/resource-exporter/pkg/numatopo"
+)
+
+var logFlushFreq = pflag.Duration("log-flush-frequency", 5*time.Second, "Maximum number of seconds between log flushes")
+
+func getNumaTopoClient(argument *args.Argument) (*versioned.Clientset, error) {
+	config, err := args.BuildConfig(argument.KubeClientOptions)
+	if err != nil {
+		return nil, err
+	}
+
+	return versioned.NewForConfigOrDie(config), err
+}
+
+func numatopoIsExist(client *versioned.Clientset) (bool, error) {
+	hostname := os.Getenv("MY_NODE_NAME")
+	if hostname == "" {
+		return false, fmt.Errorf("get Hostname failed")
```

Comment:
Is the log support rotating? if not, the error log will occupy too much disk.

Reply:
currently， the log isn't redirected to the specific path，just use "kubectl logs XXX" to check it, So if users want it, they  need to  adapt the rotation.